internal/pkg/admission/mutating-webhook.go

```diff
@@ -34,7 +34,7 @@ func (r *Reconciler) mutatingWebhookConfiguration(
 
 	service := admissionregistrationv1.ServiceReference{
 		Namespace: r.DeploymentsNamespace,
-		Name:      constants.PolicyServerServiceName,
+		Name:      constants.PolicyServerServiceNamePrefix,
```

Comment:
This is missing the name of this specific PolicyServer resource. It should be something like

```golang
Name: constants.PolicyServerServiceNamePrefix + r.Name
```

I'm not sure about r.Name, I'm just giving you some pseudo code :)

Reply:
I was thinking of doing this in the next card https://github.com/kubewarden/kubewarden-controller/issues/57 
Since this is still done in the ClusterAdmissionPolicy loop and we won't have the policy name until we move the code to the PolicyServer loop. 
Maybe I shouldn't have introduced the prefix here. Should I leave as it was and then do it in the PolicyServer card?
Same applies to the other comments

Reply:
Sorry, I didn't realize that. You can leave that as it is, and then add the suffix into another PR